scripts/develNet/createDataset.py

```diff
@@ -148,7 +148,13 @@ def computeClusterMetadata(self, cluster, bbox):
                 'None passed as cluster - ' \
                 + 'possibly a too-small cluster passed from clusterByBBox?')
         np_cluster = np.array(cluster)
-        features = extract_cluster_features(np_cluster, bbox)
+
+        features = Features()
+        features.cluster_id = bbox.id
+        features.cls = bbox.type
+        features.cnt = cluster.shape[0]
```

Comment:
I understand removing the bbox information from the function, but I think that it still makes sense for the cluster count to be within the function.